code/modules/uplink/uplink_items.dm

```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/reverse_bear_trap
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/clumsyDNA
+	name = "Clumsy Clown DNA"
+	desc = "A DNA injector that has been loaded with the clown gene that makes people clumsy. \
+	Making someone clumsy will allow them to use clown firing pins as well as Reverse Revolvers. For a laugh try useing this on HOS to see how many times they shoot themselfs in the foot!"
+	cost = 3
+	item = /obj/item/dnainjector/clumsymut
+	restricted_roles = list("Clown")
+
+/datum/uplink_item/role_restricted/anitclumsyDNA
+	name = "Clown punishment"
+	desc = "HONK Looks like one of are angents got, got! \
+	We got a barrle of laughs for you, HONK, we got a Mime DNA injector HONK! Use this on a clown HONK to make them not clusmy HONK!"
+	cost = 1 //If Hos or warden gets a uplink they should be able to get it for allmost free
```

Comment:
cost = 1 //If Hos or warden gets an uplink they should be able to get it for almost free